src/main/resources/application-test.yml

```diff
@@ -0,0 +1,14 @@
+
+email:
+  client:
+    sender: noreply_test_erouser@erop.ierds.uk
+    allow-list-enabled: true
+    allow-list-domains:
+      - valtech.com
+      - levellingup.gov.uk
+      - softwire.com
+  content:
+    portal-base-url: https://ero.test.erop.ierds.uk
+    certificate-returned:
+      email-body-template: email-templates/certificate-returned-la-emails.html
```

Comment:
It will be clear to rename all `non-prod` email templates with name starting `non-prod-certificate-returned-emails.html`

Reply:
or a separate `non-prod` folder would be better